test/rpc-integration/test-a-to-f/response-branching.booster-dev

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
the rules now have a side-condition and the booster cannot currently handle such branching, so the booster-dev just aborts

Reply:
OK, so we should probably rename the test case to "aborting-on-side-condition", and have a different test input that triggers branching